app/jobs/cache_analytics_job.rb

```diff
@@ -398,6 +399,23 @@ def self.all_monitoree_snapshots(analytic_id, monitorees, subjur_ids)
                                                  .where_assoc_exists(:patient, isolation: workflow == 'Isolation')
                                                  .in_time_frame(time_frame)
                                                  .size,
+                        exposure_to_isolation_active: workflow == 'Isolation' ? exposure_to_isolation.monitoring_open.size : nil,
+                        exposure_to_isolation_not_active: workflow == 'Isolation' ? exposure_to_isolation.monitoring_closed
+                                                                                                         .or(exposure_to_isolation.purged)
```

Comment:
it seems like the `monitoring_closed` scope does not filter out purged monitorees so I don't think this is necessary

```suggestion

```

Reply:
Agreed. I think I included the `or` here just out of an abundance of caution, but it shouldn't be possible for a purged monitoree to have `monitoring: true`.

Reply:
@tstrass should I remove it or leave it?

Reply:
I ran a query in prod to check this and apparently there actually is a very small but nonzero number of patients where `purged: true, monitoring: true`. Hasn't happened since March, but since we don't know how those patients got in that state, I think I'd prefer to just leave in the seemingly redundant `or` clause here. @timwongj are you good with that?

Reply:
sounds good